resources/migrations/000_migrations.yaml

```diff
@@ -8339,6 +8339,23 @@ databaseChangeLog:
               WHERE base_type IN ('ArrayField', 'BigIntegerField', 'BooleanField', 'CharField', 'DateField',
                                   'DateTimeField', 'DecimalField', 'DictionaryField', 'FloatField', 'IntegerField',
                                   'TextField', 'TimeField', 'UUIDField', 'UnknownField');
+  - changeSet:
+      id: 306
+      author: howonlee
+      comment: Added 0.40.0 Track cache hits in query_execution table
+      changes:
+        - addColumn:
+            tableName: query_execution
+            columns:
+              - column:
+                  name: cache_hit
+                  type: boolean
+                  remarks: "Cache hit on query execution"
+                  constraints:
+                    nullable: true
+                    deferrable: false
+                    initiallyDeferred: false
```

Comment:
what is the reasoning behind these deferrable settings on a nullable boolean column?

Reply:
you know what? looks like this only works for fkeys in postgres and don't work in mssql and I don't understand how they work in oracle. let me just get rid of them